rpc/rpc-lib/src/rpc-clnt.c

```diff
@@ -23,14 +23,14 @@ void
 rpc_clnt_reply_deinit(struct rpc_req *req, struct mem_pool *pool);
 
 struct saved_frame *
-__saved_frames_get_timedout(struct saved_frames *frames, uint32_t timeout,
-                            struct timeval *current)
+__saved_frames_get_timedout(struct saved_frames *frames, time_t timeout,
+                            time_t current)
 {
     struct saved_frame *bailout_frame = NULL, *tmp = NULL;
 
     if (!list_empty(&frames->sf.list)) {
         tmp = list_entry(frames->sf.list.next, typeof(*tmp), list);
-        if ((tmp->saved_at.tv_sec + timeout) <= current->tv_sec) {
+        if ((tmp->saved_at + timeout) <= current) {
```

Comment:
Instead of passing both current and timeout, can't we just pass (current - timeout) and do a single comparison here, something like:
```
if ((tmp->saved_at <= current_minus_timeout) {
```

Reply:
OK